executor/runtime/docker/docker_linux.go

```diff
@@ -150,6 +145,11 @@ func setupSystemServices(parentCtx context.Context, c runtimeTypes.Container, cf
 			continue
 		}
 
+		if svc.Volumes != nil && svc.ContainerName == "" {
```

Comment:
Would it be more natural to say:

   if svc.Image != "" && svc.ContainerName == ""

Who cares if it has volumes, really.
Sure in the cases recently they happened to have volumes.
But the more important thing is if it had an *image*.
Cause if it had an image, but no container, it means we can't continue.

Reply:
Yes, yes it would, thank you.